### DISCUSSION TITLE
Display for support for Touchmode and Marlin Mode

### DISCUSSION THREAD

**Discussion by Infactionfreddy** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2327] [Ideas]
Hello dears, I recently built a 3D printer with a BTT TFT 70 and use it mainly in touch mode. I noticed that there are a few functions that work 100% (led event and heating up(M190,M109), remoteprinting start over marlin, pause and stop button at remoteprinting) and am currently working on display support for Marlin on extui. I wanted to ask what you think about it and if there are people who want to help me.
A few short words about me, I'm Frederic, I live in Berlin and I'm studying electronics in the field of communication electronics.

**Comment by oldman4U** [comment]
Hi Frederic. It is open source. So feel free to add new features and bug fixes whenever you believe they could help the community.

Danke

**Reply by Infactionfreddy** [reply]
Hey oldman4u, gives a code documentation on how the structures are build ?

**Reply by Infactionfreddy** [reply]
And how can i change the command behind the icon in touch mode?

**Reply by oldman4U** [reply]
There is no documentation but there have been many developers before starting from scratch without it.

**Reply by Infactionfreddy** [reply]
I'm not finished yet, but I have an idea how to implement it.  Should I open a new pull request with a short description?
